src/parse/matches/arg_matches.rs

```diff
@@ -300,6 +302,178 @@ impl ArgMatches {
         })
     }
 
+    /// Gets the value of a specific argument (i.e. an argument that takes an additional
+    /// value at runtime) and then converts it into the result type using [`std::str::FromStr`].
+    ///
```

Comment:
No mention about function returning error.

> If either the value is not present or parsing failed, returns Err

Reply:
Isn't it obvious from the return type?

Reply:
You wrote similar message for all the below docs but not this one. Just wasn't consistent.

Reply:
I believe the second abstract covers it.